sqlglot/tokens.py

```diff
@@ -168,6 +168,10 @@ def number(cls, number):
     def string(cls, string):
         return cls(TokenType.STRING, string)
 
+    @classmethod
+    def identifier(cls, identifier):
```

Comment:
Is this being used anywhere?

Reply:
not internally no, these methods are used externally